integreat_cms/cms/templates/pois/poi_list_row.html

```diff
@@ -3,7 +3,10 @@
 {% load content_filters %}
 
 <tr class="border-t border-gray-200 hover:bg-gray-100 text-gray-800">
-	<td class="pr-2 pl-4">
+	<td class="py-3 pl-2 text-gray-800 lang-grid">
+        <input type="checkbox" name="selected_ids[]" value="{{ item.id }}" class="bulk-select-item">
```

Comment:
```suggestion
        <input type="checkbox" name="selected_ids[]" value="{{ poi.id }}" class="bulk-select-item">
```
`item` was just a placeholder, it has to be adapted to the object you're using in each list.

Reply:
This is not resolved yet